pkg/kubelet/metrics/metrics.go

```diff
@@ -67,8 +67,8 @@ const (
 	KubeletConfigKeyLabelKey      = "node_config_kubelet_key"
 
 	// Metrics keys for RuntimeClass
-	RunPodSandboxLatenciesKey = "run_podsandbox_latencies"
-	RunPodSandboxErrorsKey    = "run_podsandbox_errors"
+	RunPodSandboxLatenciesKey = "run_podsandbox_latencies_seconds"
```

Comment:
typically "duration" is preferred rather than "latencies"

Reply:
Fixed it. Thanks!